pom.xml

```diff
@@ -10,6 +10,8 @@
     <packaging>pom</packaging>
 
     <properties>
+        <project.version>0.0.0${snapshotSuffix}</project.version>
```

Comment:
Do we have to remember to update this version number when we make a change? I assume yes. Might be worth an entry in the readme 🤔